collect_app/src/test/java/org/odk/collect/android/formentry/FormEntryViewModelTest.java

```diff
@@ -69,7 +69,7 @@ public void setup() {
     public void refresh_whenEventIsBeginningOfForm_stepsForwards() {
         formController.setCurrentEvent(FormEntryController.EVENT_BEGINNING_OF_FORM);
 
-        viewModel.refresh();
+        viewModel.refreshSync();
```

Comment:
We can still use `refresh` in tests as we're flushing the `Scheduler` after.

Reply:
Done.